packages/tds-migration/src/main/resources/db/data/R__project.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO project
+    (id, name, description, created_on, updated_on, deleted_on)
+VALUES
+    (1, 'ASKEM Epi Demo Project', 'ASKEM Epi Demo Project', '2023-01-01 00:00:00', '2023-01-01 00:00:00', NULL)
+ON CONFLICT DO NOTHING;
+INSERT INTO project
+    (id, name, description, created_on, updated_on, deleted_on)
+VALUES
+    (2, 'ASKEM Climate Demo Project', 'ASKEM Climate Demo Project', '2023-01-01 00:00:00', '2023-01-01 00:00:00', NULL)
+ON CONFLICT DO NOTHING;
```

Comment:
You can bundle into one `INSERT` call.
```suggestion
INSERT INTO project
    (id, name, description, created_on, updated_on, deleted_on)
VALUES
    (1, 'ASKEM Epi Demo', 'ASKEM Epi Demo', '2023-01-01 00:00:00', '2023-01-01 00:00:00', NULL),
    (2, 'ASKEM Climate Demo', 'ASKEM Climate Demo', '2023-01-01 00:00:00', '2023-01-01 00:00:00', NULL)
ON CONFLICT DO NOTHING;
```

Reply:
that's all or nothing. The original way, we can have 1 conflict and not the other.

Reply:
That makes sense, but to be honest, this is the only script that will ever add those projects, so either both exist or neither.
It doesn't matter, this is me being pedantic for no good reasons.